src/ipsis/course-merge/CourseMergeMergedOfferingCollectionEntity.js

```diff
@@ -55,5 +59,9 @@ export class CourseMergeMergedOfferingCollectionEntity extends BaseCollectionEnt
 
 		return performSirenAction(this._token, action, null, true);
 	}
+
+	updateEntity(entity) {
+		this._entity = entity;
+	}
```

Comment:
What do we need this for? Vlad just removed these from the `CourseMergeOfferingCollectionEntity` so I'd like to avoid bringing it back.